### DISCUSSION TITLE
Can include more packages like scipy.stats in the import whitelist?

### DISCUSSION THREAD

**Discussion by sunyan-SG** [sinaptik-ai/pandas-ai#235] [Ideas]
Can include more packages like scipy.stats in the import whitelist? To make pandasAI more useful for data analytics. thank you.

**Comment by gventuri** [comment]
Sounds interesting. What kind of use cases do you have in mind for that? Could you share some prompts you have in mind that would need `scipy.stats` so that we have a clearer understanding of some of the use cases you have in mind as we implement it!

**Comment by sunyan-SG** [comment]
Thanks for getting back to me.
Here are some use cases:

1. Add regression line with confidence intervals when plotting the data

2. Calculate p-values when comparing data of different categories

3. Assess if two factors are correlated with each other and their Pearson
coefficients

**Comment by gventuri** [comment]
@sunyan-SG, you should now be able to use it with version 0.5.1! Let me know if it works for you!
